### PR TITLE
Move ParallelNative and PureTorch to GHA

### PR DESCRIPTION
**Pull request by malfet**
Separate ParallelTBB move to https://github.com/pytorch/pytorch/pull/64193 as it requires some further investiagation

### CODE REVIEW

**Comment by malfet** [comment]
@pytorbot ciflow rerun -l ciflow/linux

**Review by janeyx99 — APPROVED** [review]
I may add logic to remove the confusion with on_pull_request

File: .github/scripts/generate_ci_workflows.py [hunk 1]
```diff
@@ -276,24 +276,44 @@ def generate_workflow_file(self, workflow_template: jinja2.Template) -> None:
...
-    #     docker_image_base=f"{DOCKER_REGISTRY}/pytorch/pytorch-linux-xenial-py3.6-gcc5.4",
-    #     test_runner_type=LINUX_CPU_TEST_RUNNER,
-    # ),
-    # CIWorkflow(
-    #     arch="linux",
-    #     build_environment="pure_torch-linux-xenial-py3.6-gcc5.4",
-    #     docker_image_base=f"{DOCKER_REGISTRY}/pytorch/pytorch-linux-xenial-py3.6-gcc5.4",
-    #     test_runner_type=LINUX_CPU_TEST_RUNNER,
-    # ),
+    #CIWorkflow(
```
Comment (janeyx99) [inline .github/scripts/generate_ci_workflows.py:279]:
```suggestion
    # CIWorkflow(
```
flake8

File: .github/scripts/generate_ci_workflows.py [hunk 2]
```diff
@@ -276,24 +276,44 @@ def generate_workflow_file(self, workflow_template: jinja2.Template) -> None:
...
+    #    build_environment="paralleltbb-linux-xenial-py3.6-gcc5.4",
+    #    docker_image_base=f"{DOCKER_REGISTRY}/pytorch/pytorch-linux-xenial-py3.6-gcc5.4",
+    #    test_runner_type=LINUX_CPU_TEST_RUNNER,
+    #    on_pull_request=True,
+    #    ciflow_config=CIFlowConfig(
+    #        enabled=True,
+    #        trigger_action_only=True,
+    #        labels={LABEL_CIFLOW_LINUX, LABEL_CIFLOW_CPU},
+    #    ),
+    #),
```
Comment (janeyx99) [inline .github/scripts/generate_ci_workflows.py:290]:
```suggestion
    # ),
```
flake8

**Review by janeyx99 — APPROVED** [review]

**Review by seemethere — APPROVED** [review]

**Pull request closed without merging**